projects/packages/my-jetpack/_inc/components/evaluation-recommendations/index.tsx

```diff
@@ -1,13 +1,46 @@
-import useRecommendationsSection from '../../data/recommendations-section/use-recommendations-section';
+import { Container, Col, Text } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import useEvaluationRecommendations from '../../data/evaluation-recommendations/use-evaluation-recommendations';
+import { JetpackModuleToProductCard } from '../product-cards-section/all';
 
 const EvaluationRecommendations: React.FC = () => {
-	const { isSectionVisible, recommendedModules } = useRecommendationsSection();
+	const { isSectionVisible, recommendedModules } = useEvaluationRecommendations();
 
 	if ( ! isSectionVisible ) {
 		return null;
 	}
 
-	return <>EvaluationRecommendations TBD: { recommendedModules.join( ', ' ) }</>;
+	return (
+		<Container horizontalGap={ 2 } horizontalSpacing={ 8 }>
+			<Col>
+				<Text variant="headline-small">
+					{ __( 'Our recommendations for you', 'jetpack-my-jetpack' ) }
+				</Text>
+			</Col>
+			<Col>
+				<Text>
+					{ __(
+						'Here are the features that will best help you protect your site:',
+						'jetpack-my-jetpack'
+					) }
+				</Text>
+			</Col>
+			<Col>
+				<Container horizontalGap={ 4 } horizontalSpacing={ 2 } fluid>
+					{ recommendedModules.map( module => {
+						const Card = JetpackModuleToProductCard[ module ];
+						return (
+							Card && (
+								<Col key={ module } lg={ 4 }>
+									<Card recommendation={ true } />
```

Comment:
I think you can just do this 🤔 
```suggestion
									<Card recommendation/>
```